File: TASK-03/go/subtask3.go
package main
import ("fmt"; "strings")

func main() {
    var n int
    fmt.Print("Enter an odd number: ")
    fmt.Scan(&n)

    // Upper part
    for i := 0; i <= n/2; i++ {
        fmt.Println(strings.Repeat(" ", n/2-i) + strings.Repeat("*", 2*i+1))
    }

    // Lower part
    for i := n/2 - 1; i >= 0; i-- {
        fmt.Println(strings.Repeat(" ", n/2-i) + strings.Repeat("*", 2*i+1))
    }
}

File: TASK-03/c/subtask3.c
#include <stdio.h>

int main() {
    int n;
    printf("Enter an odd number: ");
    scanf("%d", &n);

    // Upper part
    for (int i = 0; i <= n / 2; i++) {
        for (int j = 0; j < n / 2 - i; j++) printf(" ");
        for (int j = 0; j < 2 * i + 1; j++) printf("*");
        printf("\n");
    }

    // Lower part
    for (int i = n / 2 - 1; i >= 0; i--) {
        for (int j = 0; j < n / 2 - i; j++) printf(" ");
        for (int j = 0; j < 2 * i + 1; j++) printf("*");
